src/components/ui/dropdown-menu.tsx

```diff
@@ -0,0 +1,56 @@
+'use client';
+
+import * as React from 'react';
```

Comment:
According to the regulations, it is required not to use wildcards. Could you please make the necessary corrections?

Reply:
Ah I completely forgot about it. Thank you for the comment. I'll fix that.